### PR TITLE
Fix black script previews

### PR DESCRIPTION
**Pull request by Chaosus**
As suggested by @djrm in #18139

### CODE REVIEW

File: editor/plugins/editor_preview_plugins.cpp [hunk 1]
```diff
@@ -470,10 +470,10 @@ Ref<Texture> EditorScriptPreviewPlugin::generate(const RES &p_from) {
 	if (EditorSettings::get_singleton()->get("text_editor/theme/color_theme") == "Adaptive") {
 		Ref<Theme> tm = EditorNode::get_singleton()->get_theme_base()->get_theme();
```
Comment (akien-mga) [inline editor/plugins/editor_preview_plugins.cpp]: You could remove this line too if the proposed fix is correct.

Comment (Chaosus) [reply editor/plugins/editor_preview_plugins.cpp]: Done !

**Comment by Chaosus** [comment]
However these lines is simular to init lines....

```
	Color bg_color = EditorSettings::get_singleton()->get("text_editor/highlighting/background_color");
	Color keyword_color = EditorSettings::get_singleton()->get("text_editor/highlighting/keyword_color");
	Color text_color = EditorSettings::get_singleton()->get("text_editor/highlighting/text_color");
	Color symbol_color = EditorSettings::get_singleton()->get("text_editor/highlighting/symbol_color");

	if (EditorSettings::get_singleton()->get("text_editor/theme/color_theme") == "Adaptive") {
		bg_color = EditorSettings::get_singleton()->get("text_editor/highlighting/background_color");
		keyword_color = EditorSettings::get_singleton()->get("text_editor/highlighting/keyword_color");
		text_color = EditorSettings::get_singleton()->get("text_editor/highlighting/text_color");
		symbol_color = EditorSettings::get_singleton()->get("text_editor/highlighting/symbol_color");
	}
```

**Comment by Chaosus** [comment]
@akien-mga should I remove Adaptive section completly then ?

**Comment by djrm** [comment]
@Chaosus yes, you should remove the whole block.

**Pull request merged**